packages/firebase_admob/lib/firebase_admob.dart

```diff
@@ -161,11 +212,14 @@ class BannerAd extends MobileAd {
       ? 'ca-app-pub-3940256099942544/6300978111'
       : 'ca-app-pub-3940256099942544/2934735716';
 
+  final AdSize size;
+
   /// Create a BannerAd.
   ///
   /// A valid [adUnitId] is required.
   BannerAd({
     @required String adUnitId,
+    @required this.size,
```

Comment:
Why not give this a default value?

Reply:
There are two reasons, I guess.

- Our Android and iOS APIs don't have a default, so it's what I'm used to.
- In my mind, publishers should be consciously picking a size based on what their UI looks like. Most will pick the standard 320x50 banner anyway, but I'd like for them to think at least for a moment about it.

Reply:
I see, sounds reasonable.